src/manageiq_client/api.py

```diff
@@ -622,6 +622,11 @@ def _process_result(self, result):
             if "id" in result:
                 d["href"] = "{}/{}".format(collection._href, result["id"])
             return Entity(collection, d)
+        elif "task_href" in result:
```

Comment:
I simplified the condition as "href" is always required for ``Entity`` and "id" is optional (can be obtained later if not present in the data used for ``Entity`` instantiation).